<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mohamed Jamoussi Movie Booking</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>C:/Users/msi/Downloads/popcorn.png</normaloff>C:/Users/msi/Downloads/popcorn.png</iconset>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>681</width>
     <height>321</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="lblMovie">
      <property name="text">
       <string>One Piece Red Movie</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblMovie_2">
      <property name="text">
       <string>Jujutsu Kaisen</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblMovie_3">
      <property name="text">
       <string>Oppenheimer</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>430</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Choose the Movie</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>370</y>
     <width>681</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QRadioButton" name="radioButton">
      <property name="text">
       <string>One Piece Stampede</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="text">
       <string>Jujutsu Kaisen O</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_3">
      <property name="text">
       <string>Oppenheimer</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
